test/etl_test.py

```diff
@@ -131,3 +131,47 @@ def test_ferc1_solo_etl(datastore_fixture,
         pudl_settings_fixture,
         datapkg_bundle_name='ferc1-solo',
         clobber=True)
+
+
+class TestExcelExtractor:
+    """Verify that we can lead excel files as provided via the datastore."""
+
+    def test_excel_filename_eia860(self):
+        """Spot check eia860 extractor gets the correct excel sheet names."""
+        extractor = pudl.extract.eia860.Extractor()
+        assert extractor.excel_filename(
+            2011, "boiler_generator_assn") == "EnviroAssocY2011.xlsx"
+        assert extractor.excel_filename(
+            2016, "generator_retired") == "3_1_Generator_Y2016.xlsx"
+        assert extractor.excel_filename(
+            2018, "utility") == "1___Utility_Y2018.xlsx"
+
+    def test_excel_filename_eia923(self):
+        """Spot check eia923 extractor gets the correct excel sheet names."""
+        extractor = pudl.extract.eia923.Extractor()
+        assert extractor.excel_filename(2009, "plant_frame") == \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=5664147)